shared/common-adapters/multi-line-input.desktop.js

```diff
@@ -17,6 +17,12 @@ export default class MultiLineInput extends Component<void, Props, State> {
   _handleKeyUp (e: SyntheticEvent) {
     // $FlowIssue
     this.setState({textContent: e.target.textContent})
+    // This is a content editable text, not input.
+    // People using onChange may expect text to be in target.value
+    // but on content editable it is in target.textContent
+    // Copy this to provide a uniform api
+    // $FlowIssue
+    e.target.value = e.target.textContent
```

Comment:
The `Input` is an actual HTML Input element, but this uses `contentEditable` so you can't get the value from `target.value`, this just copies that data to make the interface uniform.